src/utils/misc-utils.ts

```diff
@@ -1,4 +1,6 @@
 export const stripPathing = (stringVal: string) => {
+    if (!stringVal) return stringVal;
```

Comment:
This function could have a single `return` statement:

```ts
export const stripPathing = (value: string) => {
  return !value
    ? value
    : value.substring( // substring config );
}
```

And this function is a prime contender for the generic `value` input argument, in my opinion.